ingest/src/main/java/com/connexta/ingest/adaptors/S3StorageAdaptor.java

```diff
@@ -42,16 +45,25 @@ public void store(StoreRequest storeRequest, String key) throws IOException {
     final String fileName = storeRequest.getFileName();
 
     LOGGER.info("Storing {} in bucket \"{}\" with key \"{}\"", fileName, s3BucketQuarantine, key);
-    s3Client.putObject(
-        PutObjectRequest.builder()
-            .bucket(s3BucketQuarantine)
-            .key(key)
-            .contentType(storeRequest.getMimeType())
-            .contentLength(storeRequest.getFileSize())
-            .metadata(ImmutableMap.of("filename", fileName))
-            .build(),
-        RequestBody.fromInputStream(
-            storeRequest.getFile().getInputStream(), storeRequest.getFile().getSize()));
+    try {
+      s3Client.putObject(
```

Comment:
`S3Client#putObject` can throw `AwsServiceException`, `SdkClientException`, and `S3Exception`. Should we be explicitly catch these three types of exceptions here and also `RuntimeException`? There are also many different extensions of these 3 Exception classes. that I think we'd want to handle differently, e.g. `NoSuchKeyException`, `NoSuchBucketException`, and `RetryableException`.

This all doesn't need to be addressed in this PR, but it would be worth at least catching those four exceptions; updating the message in the StorageException to match the javadoc of the `AwsServiceException`, `SdkClientException`, and `S3Exception` classes; and adding an itest for each.